tests/integration/network.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package integration
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/tests/workload"
+)
+
+var (
+	ErrUnableToConfirmTx = errors.New("unable to confirm transaction")
+	ErrInvalidURI        = errors.New("invalid uri")
+	ErrTxNotFound        = errors.New("tx not found")
+)
+
+const (
+	txCheckInterval = 100 * time.Millisecond
+)
+
+type Network struct {
+	uris []string
+}
+
+func (*Network) ConfirmTxs(ctx context.Context, txs []*chain.Transaction) error {
+	return instances[0].ConfirmTxs(ctx, txs)
+}
```

Comment:
This should confirm txs on the entire network. This means that we need to build, verify, and accept the block on the first block, and then issue, verify, and accept it on the rest of the instances in the network ie. `ParseBlock`, `Verify`, `Accept`.

Reply:
done